Automattic/wp-calypso
# Fix Upgrade to Business Nudge for Monthly Plans

#### Changes proposed in this Pull Request

The upsell to go to Business after purchasing Monthly Premium is broken. 3 things to look for:

1. The URL was yearly-specific and can now be monthly
2. The 7-day refund period mentioned at the bottom (twice) with accurate old/new price
3. Checkout has to work

<img width="1041" alt="Screenshot 2020-10-28 at 11 52 51" src="https://user-images.githubusercontent.com/82778/97420654-443fbc00-1914-11eb-9641-1aa67ebc3d1f.png">


#### Testing instructions

0. Make sure you're in `treatment` group for the a/b test
1. Apply D51830-code and D51847-code
2. Buy Monthly Premium and make sure the refund period and the prices are fine
3. Continue to checkout and make sure Business Monthly is there
4. Cancel the subscriptions
5. Buy the annual plan

Note: This is technically broken for 2-year plans. However, despite being broken, it works quite smoothly as it offers a transition from 2-year Premium to 1-year Business for just $120. I don't want to fix such a bug that can be a feature.
